config/config.go

```diff
@@ -65,18 +66,24 @@ type Resource struct {
 	Ingress               bool `json:"ing"`
 }
 
+// Event struct for granular config
+type Event struct {
+	Global []string `json:"string,omitempty"`
+	Create []string `json:"create,omitempty"`
+	Update []string `json:"update,omitempty"`
+	Delete []string `json:"delete,omitempty"`
+}
+
 // Config struct contains kubewatch configuration
 type Config struct {
 	// Handlers know how to send notifications to specific services.
 	Handler Handler `json:"handler"`
 
 	//Reason   []string `json:"reason"`
-
-	// Resources to watch.
-	Resource Resource `json:"resource"`
-
-	// For watching specific namespace, leave it empty for watching all.
+	Resource Resource `json:"resource,omitempty"`
```

Comment:
It seems that in your rebase+conflict resolutions you reverted the comments.

in #233 I added a feature that generates a "sample" configuration file annotated with doc comments:

```
$ kubewatch config sample
```

For this to work, the `Config` Go type (and all structs used for its value) must have meaningful comments, following the godoc rules, namely if a comment directly precedes a field declaration, it's considered a field documentation comment.

this is why the commented out `//Reason..` statement has been separated from the `Resource` field by an empty line.

Reply:
can you be specific about the change to correct this